src/main/java/ch/njol/skript/expressions/ExprVillagerLevel.java

```diff
@@ -7,38 +7,54 @@
 import ch.njol.skript.doc.Name;
 import ch.njol.skript.doc.Since;
 import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser;
+import ch.njol.util.Kleenean;
 import ch.njol.util.Math2;
 import ch.njol.util.coll.CollectionUtils;
 import org.bukkit.entity.LivingEntity;
 import org.bukkit.entity.Villager;
 import org.bukkit.event.Event;
 import org.jetbrains.annotations.Nullable;
 
-@Name("Villager Level")
+@Name("Villager Level/Experience")
 @Description({
-	"Represents the level of a villager.",
+	"Represents the level/experience of a villager.",
+	"The level will determine which trades are available to players.",
+	"Experience works along with the leveling system, determining which level the villager will move to.",
```

Comment:
some more explanation would be appreciated, or a link to a wiki or something.
ex: is it a 0 to 1 scale, a 0 to 100, 1 to 100, is it different amounts per level, etc

Reply:
Ok I updated the docs to be a little bit more detailed, i included a McWiki link for trading.
The talk about experience is a bit all over the place. Hopefully this is enough

Reply:
Wow the wiki really has nothing to say on this either. I suppose it's alright but it would be nice to do some testing just to say, like, "the experience requirement for the next profession level increases per level" or "the experience requirement is always 0 to 100" or something like that

Reply:
DANG I put the wrong wiki link... will update shortly.